hashtable/hashtable.py

```diff
@@ -7,22 +8,25 @@ def __init__(self, key, value):
         self.value = value
         self.next = None
 
-
 # Hash table can't have fewer than this many slots
 MIN_CAPACITY = 8
 
-
-class HashTable:
-    """
+"""
     A hash table that with `capacity` buckets
     that accepts string keys
 
     Implement this.
-    """
-
+"""
+# HashTable class using chaining
+class HashTable:
```

Comment:
Class implementation- W00t! Some employers will let you use one way, and then ask how you'd do it _the other_ way, so it's great that you're aware of both!